ecl/hql/hqlmanifest.hpp

```diff
@@ -21,5 +21,6 @@
 
```

Comment:
Overall, the code appears to be well-structured and follows the licensing and copyright guidelines. However, there are a few recommendations for improving the code:

1. **Comments:** It would be beneficial to include comments for the functions to explain what they do, their parameters, and their return values. This will make the code easier to understand for other developers.

```cpp 
// Adds resources from the manifest file to the archive.
extern HQL_API void addManifestResourcesToArchive(IPropertyTree *archive, const char *filename);

// Checks if the manifest file is valid.
extern HQL_API bool isManifestFileValid(const char *filename);

// Calls Jfrog with the provided parameters.
extern HQL_API void callJfrog(StringBuffer &localpath, const char *manifestFilename, IPropertyTree &item);
```

2. **Function Naming:** The function `callJfrog` is not very descriptive. Consider renaming it to something more descriptive, such as `executeJfrogCommand`.

```cpp 
extern HQL_API void executeJfrogCommand(StringBuffer &localpath, const char *manifestFilename, IPropertyTree &item);
```

3. **Error Handling:** It's not clear how errors are handled in these functions. It would be beneficial to have some error handling or at least return some information if something goes wrong. 

4. **Consistent Naming Convention:** Ensure that the naming convention is consistent. For instance, `filename` in `addManifestResourcesToArchive` and `manifestFilename` in `callJfrog` refer to the same thing. It would be best to use the same term for clarity and consistency. 

5. **Include Guards:** The include guards are properly used, ensuring that the header file is included only once in a single compilation. 

Please note that these are recommendations and the code might still work perfectly without these changes. It's always a good idea to improve code readability and maintainability, especially when working in a team.